### DISCUSSION TITLE
How to access the same db as different users

### DISCUSSION THREAD

**Discussion by KevinMyers05** [sqlitecloud/discussions#42] [Help] [answered]
I am currently working on a project for my class which requires a program that I will write to access the db on the sqlite cloud. I am trying to get another account to access it through the website to no avail. Is there any way for a separate user to access it through the site or through some other means?

**Comment by marcobambini** [comment, marked as answer]
Hello @KevinMyers05,
Project sharing is a planned feature.
It will be announced soon.
